crates/sncast/tests/docs_snippets/validation.rs

```diff
@@ -65,14 +64,6 @@ fn test_docs_snippets() {
             }
         }
 
-        // check if one of "declare", "deploy" or "invoke" is included in args
-        if args
-            .iter()
-            .any(|arg| *arg == "declare" || *arg == "deploy" || *arg == "invoke" || *arg == "run")
-        {
-            args.extend(TEST_RESOURCE_BOUNDS_FLAGS);
-        }
-
```

Comment:
Why isn't it required anymore?

Reply:
This change should was accidentally added to this PR, it should stay on https://github.com/foundry-rs/starknet-foundry/pull/3101